rfcs/QueryLevelNullability.md

```diff
@@ -79,22 +79,23 @@ GraphQL on the client. GraphQL codegen tooling exists for many platforms:
 
 ## 📜 Problem Statement
 
-The two modern languages used on mobile clients, Swift and Kotlin, are both non-null by default. By contrast, in a
-GraphQL type system, every field is nullable by default. From the [official GraphQL best practice](https://graphql.org/learn/best-practices/#nullability):
+In our experience, client developers have been frustrated that the vast majority of fields are nullable.
```

Comment:
should we rename this file `ClientControlledNullability` and re-title it per [Lee's suggestion](https://github.com/graphql/graphql-spec/issues/867#issuecomment-840807186) on the strawman? out of all the feedback we've received, seems like a low-hanging fruit with a big impact 😊 I also wouldn't be opposed to porting over the terminology/definitions he provided

Reply:
+1 on both changing the name to `ClientControlledNullability` and incorporating the definitions